### DISCUSSION TITLE
Is there a way to apply theme to a title bar?

### DISCUSSION THREAD

**Discussion by Mykoliukasas** [rdbende/Sun-Valley-ttk-theme#21] [Q&A] [answered]
Hello there!

First of all the theme looks amazing! Just got one question.

In the screenshots I can see that there's a title bar which matches the theme, however when I use the example script, or even apply this theme to my own projects it doesn't seem to work. Am I missing something? Thanks.
![example](https://user-images.githubusercontent.com/101264071/157605984-42a8efad-b5b3-447d-a56e-dbe9a35fcf23.png)

**Comment by rdbende** [comment, marked as answer]
You can't the titlebar appearance in Tkinter. I just simply made a custom Aurorae theme for each screenshot.

However, I recently found a method to force the titlebar to be dark on Windows. Here it is: https://gist.github.com/Olikonsti/879edbf69b801d8519bf25e804cec0aa
